### PR TITLE
ipatests: definitions for SSSD COPR nightly

### PR DESCRIPTION
**Pull request by ikerexxe**
Defined the tests that will be executed for SSSD's COPR nightly.

### CODE REVIEW

File: ipatests/prci_definitions/nightly_latest_sssd.yaml [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    priority: 100
+    job:
+      class: Build
+      args:
+        copr: '@sssd/nightly'
+        git_repo: '{git_repo}'
+        git_refspec: '{git_refspec}'
+        template: &ci-master-latest
+          name: freeipa/ci-master-f36
+          version: 0.0.5
```
Comment (f-trivino) [inline ipatests/prci_definitions/nightly_latest_sssd.yaml:43]: uhm, our f36 latest template version is: "0.0.7" ref: https://github.com/freeipa/freeipa/blob/master/ipatests/prci_definitions/nightly_latest.yaml#L54

Comment (ikerexxe) [reply ipatests/prci_definitions/nightly_latest_sssd.yaml]: Updated.

**Review by f-trivino — APPROVED** [review]
@ikerexxe  thanks for the nightly template, I'll add ack as soon as gating tests are passing.

**Review by flo-renaud — COMMENTED** [review]
Hi @ikerexxe 
Thanks for the PR. I would also add the test test_integration/test_smb.py as it already allowed to discover regressions in the past.

**Comment by ikerexxe** [comment]
Thanks for the feedback @flo-renaud!

I've updated the [freeipa-pr-ci2 PR](https://github.com/freeipa-pr-ci2/freeipa/pull/1822) first and once the tests run successfully there I'll update this PR.

**Comment by flo-renaud** [comment]
The test definitions are correct, as one can see in https://github.com/freeipa-pr-ci2/freeipa/pull/1822. Ack.

**Comment by flo-renaud** [comment]
master:

* 03e9139504261f043c215879a54c18a89f81c534 ipatests: definitions for SSSD COPR nightly

**Comment by flo-renaud** [comment]
@ikerexxe 
The test definitions have been merged to ipa master branch. I have opened a PR so that our scheduler launches the tests on Sunday morning (see https://github.com/freeipa/freeipa-pr-ci/pull/469), I'll keep you posted when everything is in place.

**Comment by ikerexxe** [comment]
Thank you @flo-renaud!

**Comment by flo-renaud** [comment]
@ikerexxe the tests will be launched weekly on Sunday, starting this week-end.
A FreeIPA developer is responsible for checking their status and opening tickets against freeipa / sssd upstream projects if new failures are detected, or marking the known issues with a link to the existing tickets.

**Pull request closed without merging**